.github/workflows/doctests.yml

```diff
@@ -0,0 +1,32 @@
+name: Doctests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  doctest:
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, '#skip-doctests') }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .
+    
+    - name: Run doctests
```

Comment:
I think we should add this part to documentation.yml and remove the doctest.yml file